src/LaunchDarkly.EventSource/Events/Utf8ByteSpan.cs

```diff
@@ -1,128 +0,0 @@
-﻿using System;
-using System.Text;
-
-namespace LaunchDarkly.EventSource.Events
-{
-    /// <summary>
-    /// Points to a span of UTF-8-encoded text in a buffer.
-    /// </summary>
-    /// <remarks>
-    /// This is similar to the <c>Span</c> type in .NET 5. It is used internally by
-    /// <c>EventSource</c> to store event data if the stream is using UTF-8 encoding.
-    /// If so, reading <see cref="MessageEvent.DataUtf8Bytes"/> will return the same
-    /// byte array, to avoid unnecessary copying.
-    /// </remarks>
-    public struct Utf8ByteSpan
```

Comment:
This is no longer a public type because `MessageEvent` no longer has a `DataUtf8Bytes` property. It's now in `Internal` and is used only as a convenience container for a tuple of (buffer, offset, length).